examples/main.tf

```diff
@@ -2,8 +2,8 @@ terraform {
   required_version = ">= 0.13.0"
   required_providers {
     datadome = {
-      version = "1.0.0"
-      source  = "datadome/datadome"
+      version = "0.0.1"
+      source  = "datadome.co/app/datadome"
```

Comment:
Could you explain this change for someone who didn't follow the whole "story"?

I'm a bit confused about the version number going so far back.

Reply:
@darahak-datadome : Added a comment to explain that we are retrieving the provider that was generated by the Makefile, and not the provider that is available on the public Terraform registry. The version and the location are variables set in the Makefile - it's of little use to change them.

Reply:
@Lauredg - Can the version be "local" ? or include "local"?  or does it have to be numbers ?
That would make it more explicit than an arbitrary low number (that gives the feeling of unstability)